getting-started/templates/pilot-sizing.yaml

```diff
@@ -135,6 +135,45 @@ dataframeservice:
     targetCPUUtilizationPercentage: 60
     targetMemoryUtilizationPercentage: 70
 
+  ingestion:
+    # Configuration for the pool of streams used to upload table data to storage such as S3 or
+    # Azure Blob Storage.
+    streamPool:
```

Comment:
Some of the settings your are overriding here are not documented in the [sizing-example](https://github.com/ni/install-systemlink-enterprise/blob/main/getting-started/templates/sizing-examples/data-management-sizing-example.yaml) template. Should they be? It seems like the rollout workflow is to start with this pilot template, and then to use the sizing example to help transition from the pilot-sized cluster to a cluster which can be used for initial scale testing.

Reply:
Also, should these settings be included in our sizing wiki?

Reply:
I've updated the sizing example template to include these settings with our default config applied, good call. I was wondering how folks were meant to transition from the pilot deployment.

> Also, should these settings be included in our sizing wiki?

They should! I've created a [task](https://dev.azure.com/ni/DevCentral/_workitems/edit/3132923) for adding them to the wiki.